### PR TITLE
[staging-next] pythonPackages.graphite-web: update dependencies

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
build is broken

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/graphite-web/default.nix [hunk 1]
```diff
@@ -15,8 +28,19 @@ buildPythonPackage rec {
   ];
 
   propagatedBuildInputs = [
-    django django_tagging whisper pycairo cairocffi
-    ldap memcached pytz urllib3 scandir
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/graphite-web/default.nix:19]: txamqp, gunicorn, pyparsing, whitenoise and six where addded.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 117056` at cc754317 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/30d9a6dccddbe61625b72be137947f1b">python38Packages.graphite-web</a></li></li>
    <li><a href="https://gist.github.com/9522b18cca3af9c0d9fe2a3a62fdb41d">python39Packages.graphite-web</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Please add a comment on the line above, explaining the purpose of this patch.
Near pkgs/development/python-modules/graphite-web/default.nix:27:5:
```
   |
27 |     ./update-django-tagging.patch
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/graphite-web/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/development/python-modules/graphite-web/default.nix:19]: Yes, they are required according to https://github.com/graphite-project/graphite-web/blob/1.1.7/requirements.txt.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 117056` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python38Packages.graphite-web</li>
<li>python39Packages.graphite-web</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
python38Packages.graphite-web:
<details>


warning: python-include-tests
Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
warning: no-python-tests
Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md
</details>
</details>

**Comment by dotlambda** [comment]
> Add a `checkPhase` for tests, or at least `pythonImportsCheck`.

I have no clue how https://github.com/graphite-project/graphite-web/blob/1.1.7/tox.ini#L20 works and I also don't which module I would try importing for `pythonImportsCheck`.

**Comment by rmcgibbo** [comment]
I think `pythonImportsCheck = [ "graphite" ];` will work, based on [this](https://github.com/graphite-project/graphite-web/blob/master/setup.py#L93).

**Review by rmcgibbo — APPROVED** [review]

**Pull request merged**